### ISSUE TITLE
v3 bulk sub/unsub

### ISSUE THREAD

**Issue by karpet** [jdigory/p5-Mail-Chimp3#1]
Mailchimp has announced some new endpoints for v3 bulk subscribe/unsubscribe actions, outside of the batch API.

http://developer.mailchimp.com/documentation/mailchimp/reference/lists/#create-post_lists_list_id

Thoughts on how best to support these via commands? Happy to submit a PR with guidance on naming, etc.

**Comment by jdigory** [comment]
You should be able to use the existing `list` method (line 799), and then just pass it the proper parameters.

**Comment by karpet** [comment]
You are suggesting this?
```perl
$chimp->list(
  method => 'POST',
  list_id => $list_id,
  # etc
);
```
That does not seem to work. The `method` argument is interpreted as a query parameter.

This does:
```diff
diff --git a/lib/Mail/Chimp3.pm b/lib/Mail/Chimp3.pm
index 8f25026..5d103b7 100644
--- a/lib/Mail/Chimp3.pm
+++ b/lib/Mail/Chimp3.pm
@@ -827,6 +827,11 @@ has endpoints => (
                 method => 'DELETE',
                 path   => 'lists/:list_id',
             },
+            batch_list => {
+                method => 'POST',
+                path   => 'lists/:list_id',
+                mandatory => ['members'],
+            },
             abuse_reports        => { path => 'lists/:list_id/abuse-reports' },
             abuse_report         => { path => 'lists/:list_id/abuse-reports/:report_id' },
             list_activity        => { path => 'lists/:list_id/activity' },
```

Happy to submit as a PR if that looks ok to you.

**Comment by jdigory** [comment]
Ah, sorry -- yes, you are right, there is no support for the POST to lists/:list_id right now, but your patch looks great. I will happily include it if you want to make a PR. Thanks!

**Comment by jdigory** [comment]
Closed by PR #2. Thanks!
